cms/templates/webinar_page.html

```diff
@@ -0,0 +1,49 @@
+{% extends "base.html" %}
+{% load static render_bundle image_version_url wagtailcore_tags wagtailimages_tags wagtailmetadata_tags tz %}
+
+{% block title %}{{ site_name }} | {{ page.title }}{% endblock %}
+
+{% block headercontent %}
+<div id="header"></div>
+{% render_bundle 'header' %}
+{% endblock %}
+
+{% block content %}
+
+  {% if page.banner_image %}
+    <div class="resource-banner" style="background-image:url('{% image_version_url page.banner_image "original" %}');background-size:cover">
+  {% else %}
+    <div class="resource-banner" style="background-image:url('{% static webinar_default_images|random %}');background-size:cover;">
+  {% endif %}
+    <div class="container">
+      <h1 class="text-uppercase">{{ page.title }}</h1>
+      {% if page.sub_heading %}
+        <h2>{{ page.sub_heading }}</h2>
+      {% endif %}
+    </div>
+  </div>
+
+<div class="container resource-container">
+  <div class="row">
+    <div class="col-md-12">
+      {% if page.date %}
+      <p>{{ page.formatted_date }} | {{ page.time }}</p>
+      {% endif %}
+      {% if page.action_url and page.category|lower != 'upcoming' %}
+        <div class="col-md-12 resource-video">
+          <iframe src="{{page.action_url}}" width="80%" height="500" frameborder="0" allow="autoplay" scrolling="no" allowfullscreen></iframe>
+        </div>
+      {% endif %}
+      {% if page.category|lower == 'upcoming' %}
+        <a href="{{page.action_url}}" class="read-more">REGISTER</a>
+      {% endif %}
```

Comment:
Could we also add a check to display the link only when there is page.action_url?